device-registry/src/test/scala/org/genivi/sota/device_registry/DeviceRequests.scala

```diff
@@ -49,27 +52,24 @@ trait DeviceRequests { self: ResourceSpec =>
     Put(Resource.uri(api, uuid.show), device)
 
   def createDevice(device: DeviceT)
-                  (implicit ec: ExecutionContext): HttpRequest = {
+                  (implicit ec: ExecutionContext): HttpRequest =
```

Comment:
Agreed, however, in this case the format change makes the whole file consistent in terms of style. This is a refactoring ticket after all. I can change it if you prefer though.